web/src/lib/domain-logic/routes.ts

```diff
@@ -41,6 +41,7 @@ export enum QUERIES {
   industry = "industry",
   code = "code",
   openTaxFilingsModal = "openTaxFilingsModal",
+  flow = "flow",
```

Comment:
Anytime we add a query value here, we also need to add the type definition for its allowed values in the interface QUERY_PARAMS_VALUES. This allows us to do static type checking when we route to query parameters in the code